Weekly-Store-Sales.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Weekly Store Market Sales By Region</title><!-- Latest compiled and minified CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <style>
+    .header {
+      text-align: center;font-style: italic; color:green;
+    }
+  </style>
+</head>
+<body>
+  <div class="container">
+    <div class="header">
+      <h1>Weekly Store Sales by Market</h1>
+      <u>Comparison of Medium to Large Markets.</u>
+    </div>
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Market Size</th>
+          <th>Location</th>
+          <th>Week</th>
+          <th>Sales in Thousands</th>
+        </tr> 
+      </thead>
+      <tbody>
+        <tr>
+          <td>Medium</td>
+          <td>1</td>
```

Comment:
It is a number so you would have to combine a lookup table to produce name.  I used simple dummy data set to be sure I got the basics down.